integration/cli/build.configequality.test.ts

```diff
@@ -76,95 +74,104 @@ describe("build", () => {
+          { kind: kind.Service, name: (uuid: string): string => uuid },
+          { kind: kind.Service, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.Role, name: (uuid: string): string => `${uuid}-store` },
+          { kind: kind.Deployment, name: (uuid: string): string => uuid },
+          { kind: kind.Deployment, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.MutatingWebhookConfiguration, name: (uuid: string): string => uuid },
+          { kind: kind.ValidatingWebhookConfiguration, name: (uuid: string): string => uuid },
+        ];
+
+        it.each(
+          sources.flatMap(src =>
+            resourceKinds.map(r => ({
+              source: src.source,
+              getResources: src.getResources,
+              resourceKind: r.kind,
+              name: r.name,
+            })),
+          ),
+        )(
+          "$resourceKind in $source resources should be defined",
+          ({ getResources, resourceKind, name }) => {
+            const res = resource.select(getResources(), resourceKind, name(peprUuid));
             expect(res).toBeDefined();
-          }
-        });
+          },
+        );
 
+        // --- ClusterRoleBinding tests ---
         it.each(sources)(
           "ClusterRoleBinding in $source resources should have correct roleRef and subjects",
           ({ getResources }) => {
-            const clusterRoleBinding = resource.select(
-              getResources(),
-              kind.ClusterRoleBinding,
-              peprUuid,
-            );
-            expect(clusterRoleBinding).toBeDefined();
-            expect(clusterRoleBinding.roleRef.name).toBe(peprUuid);
-            expect(clusterRoleBinding.subjects?.at(0)?.name).toBe(peprUuid);
+            const crb = resource.select(getResources(), kind.ClusterRoleBinding, peprUuid);
+            expect(crb).toBeDefined();
+            expect(crb.roleRef.name).toBe(peprUuid);
+            expect(crb.subjects?.[0]?.name).toBe(peprUuid);
           },
         );
 
+        // --- RoleBinding tests ---
         it.each(sources)(
           "RoleBinding in $source resources should have correct roleRef and subjects",
           ({ getResources }) => {
-            const roleBindingStore = resource.select(
-              getResources(),
-              kind.RoleBinding,
-              `${peprUuid}-store`,
-            );
-            expect(roleBindingStore).toBeDefined();
-            expect(roleBindingStore.roleRef.name).toBe(`${peprUuid}-store`);
-            expect(roleBindingStore.subjects?.at(0)?.name).toBe(`${peprUuid}-store`);
+            const rb = resource.select(getResources(), kind.RoleBinding, `${peprUuid}-store`);
+            expect(rb).toBeDefined();
+            expect(rb.roleRef.name).toBe(`${peprUuid}-store`);
+            expect(rb.subjects?.[0]?.name).toBe(`${peprUuid}-store`);
           },
         );
 
+        // --- Deployment (admission) ---
```

Comment:
```suggestion
```

This is already scoped within a `describe` block for `uuid`. The test name makes the purpose of the test clear.